keepalived/check/check_http.c

```diff
@@ -123,6 +123,38 @@ alloc_http_get(char *proto)
 	return http_get_chk;
 }
 
+static int
+compare_http_get_check(void *a, void *b)
+{
+	http_checker_t *old = CHECKER_DATA(a);
+	http_checker_t *new = CHECKER_DATA(b);
+	size_t n;
+	url_t *u1, *u2;
+
+	if (compare_conn_opts(CHECKER_CO(a), CHECKER_CO(b)) != 0)
+		goto err;
+	if (old->nb_get_retry != new->nb_get_retry)
+		goto err;
+	if (old->delay_before_retry != new->delay_before_retry)
+		goto err;
```

Comment:
Similar to retry, I don't think differing values of nb_get_retry and delay_before_retry make this a different checker.